README.md

```diff
@@ -1,4 +1,5 @@
 # Fresnel example notebooks
+[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/mikemhenry/fresnel-examples/feat/cache-mybinder?urlpath=lab/tree/index.ipynb)
```

Comment:
Many users find fresnel-examples embedded in the fresnel docs. Should we put a similar link in the fresnel documentation? Or should we add an individual notebook link to the bottom of every notebook?

Reply:
I had forgotten how tightly integrated these notebooks are in the documentation. I think having a link in every notebook (that then takes the user to that notebook on mybinder) is a good idea. I think if we do that, we can get rid of `index.ipynb` since the primary way users will be getting to these on mybinder will be links in the doc, and if they use a link on this repo that drops them into the root, we can assume they know where they want to go.

Reply:
Sounds like a great idea. I organize the directory structure of the examples so it is navigable with no other information.